### PR TITLE
bs-platform 5.0.1-beta.1

### PR DESCRIPTION
**Pull request by cknitt**
bs-platform 5.0.1-beta.1 with `"react-jsx": 3` and `[@bs.inline]` support.

### CODE REVIEW

File: bs-react-native-next/src/apis/ActionSheetIOS.bs.js [hunk 1]
```diff
@@ -1,2 +1 @@
-// Generated by BUCKLESCRIPT VERSION 5.0.0, PLEASE EDIT WITH CARE
```
Comment (MoOx) [inline bs-react-native-next/src/apis/ActionSheetIOS.bs.js]: I guess that's new? Is this cause the name is .bs.js or does bsb just dropping this "feature"?

File: bs-react-native-next/src/apis/NativeModules.bs.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-// Generated by BUCKLESCRIPT VERSION 5.0.0, PLEASE EDIT WITH CARE
+
```
Comment (MoOx) [inline bs-react-native-next/src/apis/NativeModules.bs.js]: why is there new line in some cases & some cases not?

File: bs-react-native-next/src/apis/Platform.rei [hunk 3]
```diff
@@ -0,0 +1,9 @@
+type os = pri string;
```
Comment (MoOx) [inline bs-react-native-next/src/apis/Platform.rei]: what does `pri string` means?

File: bs-react-native-next/src/apis/ActionSheetIOS.bs.js (hunk 1 above)
Comment (cknitt) [reply bs-react-native-next/src/apis/ActionSheetIOS.bs.js]: I added `-bs-no-version-header` to bsconfig.json. ;-)

Comment (MoOx) [reply bs-react-native-next/src/apis/ActionSheetIOS.bs.js]: oh I missed that :D

File: bs-react-native-next/src/apis/NativeModules.bs.js (hunk 2 above)
Comment (cknitt) [reply bs-react-native-next/src/apis/NativeModules.bs.js]: Different in bs-platform 5.0.1 for some reason unknown to me.

File: bs-react-native-next/src/apis/Platform.rei (hunk 3 above)
Comment (cknitt) [reply bs-react-native-next/src/apis/Platform.rei]: `private` in OCaml (as suggested by Hongbo in https://github.com/BuckleScript/bucklescript/issues/3472#issuecomment-480190956) becomes `pri` in Reason.

Just `type os;` alone won't work as the inline constants are of type `string`.

Comment (MoOx) [reply bs-react-native-next/src/apis/Platform.rei]: Thanks for the explanation.

File: bs-react-native-next/bsconfig.json [hunk 4]
```diff
@@ -16,6 +16,9 @@
   "ppx-flags": [
     "./node_modules/reason-react/_build/default/jsx_ppx/reactjs_jsx_ppx_v3.exe"
   ],
-  "bsc-flags": ["-bs-super-errors", "-warn-error @a", "-w -41..42-102"],
```
Comment (cknitt) [inline bs-react-native-next/bsconfig.json]: `-bs-super-errors` is the default now AFAIK, so I removed it. I added `-bs-no-version-header` to get rid of the version headers in future diffs.

Comment (MoOx) [reply bs-react-native-next/bsconfig.json]: Make sense!

**Comment by MoOx** [comment]
Feel free to merge, will fix CI this evening or tomorrow morning (fixed on the website PR - well, failing but for another reason - I have that fixed locally)

**Pull request merged**